player/playloop.c

```diff
@@ -122,7 +122,7 @@ static void mp_process_input(struct MPContext *mpctx)
 }
 
 // Process any queued option callbacks.
-static void handle_option_callbacks(struct MPContext *mpctx)
+void handle_option_callbacks(struct MPContext *mpctx)
```

Comment:
if no longer static, should have `mp_`

```suggestion
void mp_handle_option_callbacks(struct MPContext *mpctx)
```

Reply:
Most non-static stuff in playloop doesn't have `mp` though.

Reply:
¯⁠\⁠\_⁠(⁠ツ⁠)⁠_⁠/⁠¯ at least new code can be clean, we shouldn't have not prefixed functions in global scope imho.